src/source/app/kvsapp.c

```diff
@@ -513,38 +517,56 @@ static int updateAndVerifyRestfulReqParameters(KvsApp_t *pKvs)
 
 static int setupDataEndpoint(KvsApp_t *pKvs)
 {
-    int res = ERRNO_NONE;
+    int xRes = KVS_ERRNO_NONE;
```

Comment:
Not critical but I think `x` prefix was used as non stdint types.

Reply:
Yes, you are right. I'll make the change in this PR since this is a big PR.